apps/backend/src/app/common/dto/pagination-query.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { IsOptional, IsPositive } from 'class-validator';
+
+export class PaginationQueryDto {
+  @IsOptional()
+  @IsPositive()
+  limit: number;
```

Comment:
I think we should limit the `limit` value sice someone can fetch milion of items in single request